templates/codelists/version.html

```diff
@@ -70,8 +70,29 @@ <h3>{{ codelist.name }}</h3>
         Publish version
       </button>
       {% endif %}
-      {% endif %}
     </div>
+
+    <hr />
+
+    <h6>Versions</h6>
+    <ul class="pl-3">
+      {% for version in versions %}
```

Comment:
How are these ordered?

Reply:
The view is doing `order_by("-version_str")`

Reply:
Totally missed that.